lib/services/northBound/contextServer.js

```diff
@@ -957,6 +1177,79 @@ function handleNotificationNgsi1(req, res, next) {
     }
 }
 
+function handleNotificationNgsiLD(req, res, next) {
```

Comment:
Missing Javadoc ?

Reply:
Fixed c70d9ed

Reply:
Fixed and Propagated (contextServer isn't the focus of Measures) : a5c650a